make/modules/jdk.incubator.vector/Lib.gmk

```diff
@@ -27,7 +27,7 @@ include LibCommon.gmk
 
 ################################################################################
 
-ifeq ($(call isTargetOs, linux windows)+$(call isTargetCpu, x86_64), true+true)
+ifeq ($(call isTargetOs, linux windows)+$(call isTargetCpu, x86_64)+$(INCLUDE_COMPILER2), true+true+true)
```

Comment:
Instead of introducing a new exported variable from spec.gmk, you can simply check for the JVM feature like this:

$(call check-jvm-feature, compiler2)

This macro will return "true" or "false" so can be used just like $(INCLUDE_COMPILER2) in the line above. Due to line length I would probably put it in a nested ifeq statement though.